scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -119,11 +122,15 @@
 
     private static final String SESSION_ID_PARAMETER_NAME = "sessionId";
 
+    private static final String ERROR_MESSAGE = "errorMessage";
+
+    private static final String JSON_ERROR = "JSON parse ERROR";
+
     private static final String ERROR_MESSAGE_REGEX = "\"errorMessage\":\"(.*)\",\"stackTrace\"";
 
-    private static final int width = 600;
+    private static final int widthWindows = 600;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)